doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -17,28 +17,47 @@
   * [3.8.2 Resiliency Measurements](#3.8.2)
 * [3.9 NFVI Test Cases](#3.9)
 
+
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-> Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
+The objective of this chapter is to describe the requirements on NFVI test cases as derived from the reference model and architecture for the LFN-based compliance program. This set of requirements eventually determines the scope of the compliance program and the corresponding list of test cases included in the complaince program.  In particular, this chapter extends the generic list of NFVI test case requirements which is provided in Chapter 8.2.4 of the reference model.
```

Comment:
on NFVI test cases -> for NFVI test cases.

Reply:
Done. Updated. Resolved/Closed.